LaTeX/DocStrip.sublime-syntax

```diff
@@ -0,0 +1,98 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/syntax.html
+# https://www.texlive.info/CTAN/macros/latex/base/docstrip.pdf
+name: TeX (DocStrip)
+scope: source.tex.docstrip
+version: 1
+extends: Packages/LaTeX/TeX.sublime-syntax
+
+file_extensions:
+  - ins
+
+contexts:
+  main:
+    - meta_prepend: true
+    - include: docstrip-preamble
+    - include: docstrip-config
+    - include: docstrip-user-io
+    - include: docstrip-keywords
+    - include: docstrip-constants
+
+  controls:
+    - meta_append: true
+    - match: (\\)endbatchfile{{endcs}}
+      scope: keyword.control.tex.docstrip
+      captures:
+        1: punctuation.definition.backslash.tex
+    - match: (\\)(?:batch)?input{{endcs}}
+      scope: meta.function.input.tex keyword.control.input.tex
+      captures:
+        1: punctuation.definition.backslash.tex
+    - match: (\\)ifToplevel{{endcs}}
+      scope: keyword.control.conditional.tex.docstrip
+      captures:
+        1: punctuation.definition.backslash.tex
+
+  docstrip-preamble:
+    - match: (\\)(?:declare)?(?:pre|post)amble{{endcs}}
+      scope: keyword.context.block.tex.docstrip
+      captures:
+        1: punctuation.definition.backslash.tex
+      push: preamble-content
+
+
+  preamble-content:
```

Comment:
Just noticed ...

1. Are the 2 empty lines of intent?
2. I'd suggest to prepend `docstrip-` to indicate it belongs to `docstring-preamble` context and being special docstrip context. Or is `preamble-content` something which may appear in other TeX dialects and may be moved to TeX at some point?

```suggestion
      push: docstrip-preamble-content

  docstrip-preamble-content:
```

Reply:
well, LaTeX has the concept of a "preamble", but it means something entirely different. So prefixing with docstrip definitely makes sense